dts/arm/cc3220sf_launchxl.dts

```diff
@@ -22,3 +23,8 @@
 	status = "ok";
 	current-speed = <115200>;
 };
+
+&i2c0 {
+	status = "ok";
+	clock-frequency = <I2C_BITRATE_FAST>;
```

Comment:
This one is a bit tricky. While I2C module may support the fast speed mode this DTS file is written for an evaluation board. User may choose to attach a standard speed I2C device. I think it's safer to use I2C_BITRATE_STANDARD. If required the I2C module can be later reconfigured to run at a higher speed.

Reply:
Understand the reasoning behind this request, but would like to keep I2C_BITRATE_FAST here for the following reasons:

* A probable sensors board one would attach to the LaunchXL board is the BOOSTXL-SENSORS booster pack (http://www.ti.com/lit/ug/slau666a/slau666a.pdf); looking at the examples source code in TI Resource Explorer, every sensor example, without exception, sets the rate to 400 kHz when initializing the I2C driver.
* This speed also matches the SimpleLink SDK sensor samples, which is the default rate for the onboard sensors;
* It would be nice for the Zephyr apps/sensor drivers to work with the best rate supported by the hardware, without forcing a reconfiguration call;
* For example, the Zephyr samples/sensor/thermometer sample should (assuming CONFIG_CC32XX_I2C=y) work at the best I2C rate without having to modify the source code specifically for this board.

Reply:
The last two points do not sound very convincing, but the first two do. Maybe we should update board documentation mentioning that the I2C driver is configured in Fast Mode at boot?

Reply:
Updated board doc.